mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/VerifiedKycExchangeRequestDto.java

```diff
@@ -22,4 +22,5 @@ public class VerifiedKycExchangeRequestDto{
     private Map<String,Object> acceptedClaims;
```

Comment:
We do not have the field "respType" in the mock-identity-system. 

Refs -> https://github.com/mosip/esignet-mock-services/blob/develop/mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/KycExchangeRequestDtoV2.java